docs/built-in-functions.rst

```diff
@@ -545,6 +545,136 @@ Math
         >>> ExampleContract.foo(11, 2, 5)
         2
 
+.. py:function:: unsafe_add(x: integer, y: integer) -> integer
+
+    Add ``x`` and ``y``, without checking for overflow. ``x`` and ``y`` must both be integers of the same type. If the result exceeds the bounds of the input type, it will be wrapped.
+
+    .. code-block:: python
+
+        @external
+        @view
+        def foo(x: uint8, y: uint8) -> uint256:
+            return unsafe_add(x, y)
+
+        @external
+        @view
+        def bar(x: int8, y: int8) -> uint256:
+            return unsafe_add(x, y)
+
+
+    .. code-block:: python
+
+        >>> ExampleContract.foo(1, 1)
+        2
+
+        >>> ExampleContract.foo(255, 255)
+        254
+
+        >>> ExampleContract.bar(127, 127)
+        -2
```

Comment:
Is this correct behavior? Shouldnt it return an unsigned integer?

Reply:
bar takes two int8 and returns an int8